File: New/Hacker Rank/Assignment 1/py/Scripts/The Hurdle Race.py
def hurdleRace(height,n,k):
    maximum,dif=height[0],0
    for i in range(n):
        if height[i]>maximum:
            maximum=height[i]
    if maximum>k:
        dif=maximum-k
    return dif

n,k=int(input()),int(input())
height=list()
for i in range(n):
    height.append(int(input()))
print(hurdleRace(height,n,k))

File: New/Hacker Rank/Assignment 1/cpp/The Hurdle Race.cpp
//The Hurdle Race
#include<iostream>
using namespace std;
int hurdleRace(int height[],int n,int k)
{
    int maximum=height[0],dif=0;
    for(int i=0;i<n;i++)
    {
        if(height[i]>maximum)
        {
            maximum=height[i];
        }
    }
    if(maximum>k)
    {
        dif=maximum-k;
    }
    return dif;
}
int main()
{
    int n=0,k=0;
    cin>>n>>k;
    int height[n];
    if((n>=1&&n<=100)&&(k>=1&&k<=100))
    {
        for(int i=0;i<n;i++)
        {
            cin>>height[i];
        }
        cout<<hurdleRace(height,n,k);
    }

    return 0;
}
